apps/openssl.c

```diff
@@ -442,6 +442,8 @@ typedef enum HELP_CHOICE {
 } HELP_CHOICE;
 
 const OPTIONS help_options[] = {
+    {OPT_HELP_STR, OPT_hHELP, '-', "Usage: help [options]\n"},
+    {OPT_HELP_STR, OPT_hHELP, '-', "       help [command]\n"},
```

Comment:
Is duplication intentional?

Reply:
Yes. That's the way I figured out to make two lines

Reply:
Look at other uses of OPT_HELP_STR, they all use 1 not OPT_hHELP

Reply:
yes, but use 1 not OPT_hHELP the way all other instances of OPT_HELP_STR do.

Reply:
Okie, done